backend/src/errors/index.ts

```diff
@@ -9,10 +9,15 @@ const headers = {
 	'content-type': 'application/json',
 } as const
 
-function generateErrorResponse(error: string, status: number, errorDescription?: string): Response {
-	const res: ErrorResponse = {
-		error: `${error}. ` + 'If the problem persists please contact your instance administrator.',
-		...(errorDescription ? { error_description: errorDescription } : {}),
+function generateErrorResponse(error: string | Error, status: number, errorDescription?: string): Response {
+	let res: Error | ErrorResponse
+	if (error instanceof Error) {
+		res = error
```

Comment:
mh.... yeah I agree that this is not super elegant, but what would you prefer?

keeping it as is and then changing the `internalServerError` method to something like this?
```ts
export function internalServerError(error: Error): Response {
	return generateErrorResponse(error.message, 500, error.stack)
}
```?
so that `error.stack` becomes the `errorDescription`? or ignoring the stack completely and just using the error message?

___

regarding the console.log, is that necessary? 😢

Reply:
I prefer to log the error message/stack so that you can see it in the dashboard (in theory) and don't leak it to the eyeball.

Reply:
I've updated the code, please let me know if the latest change is ok 🙂